package.json

```diff
@@ -14,18 +14,23 @@
     "@angular/cdk": "^13.3.9",
     "@angular/common": "^14.3.0",
     "@angular/compiler": "^14.3.0",
-    "@angular/core": "^14.3.0",
+    "@angular/core": "^14.0.0",
```

Comment:
Is there any problem with version 14.3.0?

Reply:
NgRx store was giving issues while installing, basically i had to find a compatible version with angular core.

It was some error "root has 14.3.0 

But requirement is not meeting from NgRx store version."

I can replicate that version if you need to see.

Reply:
no, it can stay. I just want to know :)

Reply:
> no, it can stay. I just want to know :)

Sure i will replicate this and show it to you with a screenshot.

Reply:
openai-chatgpt3-clone (feature/persistentChat) $ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: open-ai-web@0.0.0
npm ERR! Found: @angular/core@14.3.0
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"^14.3.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"^15.0.0" from @ngrx/store@15.4.0
npm ERR! node_modules/@ngrx/store
npm ERR!   @ngrx/store@"^15.4.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/mehulshrinivastava/.npm/_logs/2023-04-14T10_06_55_570Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mehulshrinivastava/.npm/_logs/2023-04-14T10_06_55_570Z-debug-0.log

Error was something like this, i changed things from before when they were coming, seems like i was trying to install latest of NgRx/Store but it was not compatible with angular/core version, i read they have to point to same version number like 15.x.x == 15.x.x, correct me if i am wrong.